server/chat-commands/info.ts

```diff
@@ -517,30 +522,36 @@ const commands = {
 			if (newTarget.isInexact && !i) {
 				buffer = `No Pok\u00e9mon, item, move, ability or nature named '${target}' was found${Dex.gen > dex.gen ? ` in Gen ${dex.gen}` : ""}. Showing the data of '${newTargets[0].name}' instead.\n`;
 			}
-			/** @type {AnyObject} */
-			let details = null;
+			let details: {[k: string]: string} = {};
 			switch (newTarget.searchType) {
 			case 'nature':
 				const nature = Dex.getNature(newTarget.name);
-				buffer += "" + nature.name + " nature: ";
+				buffer += `${nature.name} nature: `;
 				if (nature.plus) {
-					const statNames = {'atk': "Attack", 'def': "Defense", 'spa': "Special Attack", 'spd': "Special Defense", 'spe': "Speed"};
-					buffer += "+10% " + statNames[nature.plus] + ", -10% " + statNames[nature.minus] + ".";
+					const statNames: {[k in StatNameExceptHP]: string} = {
```

Comment:
```suggestion
					const statNames = {
```

This annotation doesn't actually add anything...

Reply:
Have you tested this?

Reply:
No. What error comes up?

Reply:
Nothing, I had it confused with another line